src/hotspot/share/runtime/mutexLocker.cpp

```diff
@@ -136,6 +136,7 @@ Mutex*   SharedDecoder_lock           = nullptr;
 Mutex*   DCmdFactory_lock             = nullptr;
 Mutex*   NMTQuery_lock                = nullptr;
 Mutex*   NMTCompilationCostHistory_lock = nullptr;
+Mutex*   NMT_lock                     = nullptr;
```

Comment:
Sorry just noticed the other NMT locks. I think we need a more descriptive name than just plain NMT_lock so the usage is more clear.